ledger/partial/ptrie/partialTrie_test.go

```diff
@@ -293,7 +292,9 @@ func TestRandomProofs(t *testing.T) {
 		withForest(t, pathByteSize, experimentRep+1, func(t *testing.T, f *mtrie.Forest) {
 
 			// generate some random paths and payloads
-			rand.Seed(time.Now().UnixNano())
+			seed := int64(0x16695622bc383718) //time.Now().UnixNano()
+			rand.Seed(seed)
+			t.Logf("rand seed is %x", seed)
```

Comment:
I guess we should change back this to randomized so we can cover more cases.

Reply:
Good catch, that's the debugging leftover :)